ion/src/main/java/com/fasterxml/jackson/dataformat/ion/IonGenerator.java

```diff
@@ -66,6 +66,13 @@ public enum Feature implements FormatFeature // since 2.12
          * @since 2.12
          */
         USE_NATIVE_TYPE_ID(true),
+
+        /**
+         * Whether to use the Ion 1.1 encoding (true) or the Ion 1.0 encoding (false). Disabled by default.
+         *
+         * @since 2.18 // TODO change according to the minor version this lands in.
+         */
+        ION_VERSION_1_1(false),
```

Comment:
Is it possible to set something up so that instead of it being `true`/`false`, we have a "feature" that can accept an Ion Version enum?

Reply:
That's what I'd prefer too (see my comment in the PR description), but I haven't seen an example of that yet within Jackson. These enums implement a Jackson core interface so that they can be used in various places in that code, and so far I've only seen them set by booleans. I'll keep looking though.